src/native/clr/include/runtime-base/jni-wrappers.hh

```diff
@@ -0,0 +1,209 @@
+#pragma once
```

Comment:
We don't want to share code like this across MonoVM & CoreCLR? I worry about future bug fixing and duplication.

Reply:
I want to share some code, yes. This actually might be a good candidate for sharing. I'll update the PR shortly.